src/Tizen.NUI/src/public/BaseComponents/TextEditor.cs

```diff
@@ -1686,9 +1686,10 @@ private int InternalVerticalScrollPosition
         }
 
         /// <summary>
+        /// PrimaryCursorPosition property.<br />
         /// Specify primary cursor (caret) position in text control.
         /// </summary>
```

Comment:
Having an information about the result of setting it to -1, or generally negative numbers would be nice.
`<remarks>` is the section to put such information.

Reply:
@wiertel , The suggestion has been applied. Could you check it , please.